src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -67,23 +68,24 @@ class ThemeColorsEditor extends React.Component {
             return [
               <li
                 key={`dt-${idx}`}
-                className={classnames(name, "color", { selected: selectedColor === name })}
+                className={classnames(name, "color", {
+                  selected: selectedColor === name
+                })}
                 onClick={ev => this.handleClick(ev, name)}
               >
-                <span className="color__swatch" style={{ backgroundColor: colorToCSS(color) }} title={colorLabels[name]} />
-                <span className="color__label" title={colorLabels[name]} >{colorLabels[name]}</span>
+                <span
+                  className="color__swatch"
+                  style={{ backgroundColor: colorToCSS(color) }}
+                  title={colorLabels[name]}
+                />
+                <span className="color__label" title={colorLabels[name]}>
+                  {colorLabels[name]}
+                </span>
                 <span className="color__picker">
                   <SketchPicker
-                    color={{ h: color.h, s: color.s, l: color.l, a: color.a * 0.01 }}
+                    color={color}
                     disableAlpha={!colorsWithAlpha.includes(name)}
-                    onChangeComplete={({ hsl: { h, s, l, a } }) => {
```

Comment:
All of this was super awkward and should have been refactored out awhile ago. Switching to RGBA seems to make this simpler.